src/posthog-core.js

```diff
@@ -290,7 +300,16 @@ PostHogLib.prototype._loaded = function () {
     // this happens after so a user can call identify in
     // the loaded callback
     if (this.get_config('capture_pageview')) {
-        this.capture('$pageview', {}, { send_instantly: true })
+        const props = {}
+        if (this.get_config('capture_performance')) {
+            // TRICKY: without the JSON stringing and parsing less perf data is sent
+            props.performance = {
```

Comment:
Problem: Posthog kind of sucks at querying nested values right now.

Reply:
Yep, I didn't think it was right to do processing in the SDK. So my intention is to pair this with a plugin. We'll keep the "raw" performance data in case it is useful in future but use it to add useful queryable fields on ingestion

Reply:
That sounds like the feature is pretty alpha-ish. Q: Should we make the config value indicate that somehow (e.g. underscore it)?

Reply:
Definitely alpha :) I'm crossing the river by feeling the stones.